app/views/comments/_form.html.erb

```diff
@@ -23,7 +23,7 @@
 
   <div class="field">
     <%= form.label :dependency_id %>
-    <%= form.text_field :dependency_id %>
+    <%= form.select :dependency_id, Dependency.all.collect {|d| [ d.name, d.id ] } %>
```

Comment:
I believe there's a Rails helper for that.

Reply:
* Line is too long;
* I'd favor `#map` over `#collect` (but as said there might be an helper to
  remove/simplify this part);
* Does our coding style allow this syntax for the block? Shouldn't this be:
  ``` ruby
  { |d| [d.name, d.id] }
  ```